configure

```diff
@@ -2198,6 +2198,18 @@ check_looptools() {
 
    check_looptools_incl
    check_looptools_libs
+   case "$operating_system" in
```

Comment:
I think it would be better to move this `case` statement into the `check_looptools_libs` function, as this is the function which is supposed to check for the necessary libraries.

Reply:
yes, of course, sorry for this incident.

Reply:
No worries. :)